source/documentation/values/calculations.html.md.erb

```diff
@@ -210,12 +233,34 @@ combined at build time, so `min(12px % 10, 10%)` will throw an error.
   from, or compared to numbers with units. `min()` and `max()` are different,
   though: for backwards-compatibility with the global Sass `min()` and `max()`
   functions which allow unit/unitless mixing for historical reasons, these units
-  can be mixed as long as they're contained directly within a `min()` or `max()`
+  can be mixed as long as they're contained directly within a `min()` or `max()` 
   calculation.
+  For instance, `sin($number: 4)` will throw an error, as `sin()` was 
+  never a global Sass function.
 <% end %>
 
 <!-- TODO(nweiz): auto-generate this CSS once we're compiling with Dart Sass -->
 
+### [`min()`] and [`max()`]
```

Comment:
As a rule of thumb, a section of documentation should always have some amount of prose, in addition to any callouts or examples it contains. Otherwise, it's a sign that that heading isn't actually a useful division, and its contents should be part of the parent section instead.

Reply:
Got it, I added a sentence after the title, thanks! :)